uber/site_sections/preregistration.py

```diff
@@ -361,7 +361,6 @@ def process_free_prereg(self, session):
 
     @credit_card
     def prereg_payment(self, session, payment_id=None, stripeToken=None):
-        # e0953b23-cf11-4959-8f92-bf794fdaf5c7
```

Comment:
What was this line?  Did this get put here by mistake in a past commit while debugging or something?

Reply:
Yea, to be honest I have no idea what this is or why I put it here. It's definitely not something we need now, though.